sound/soc/intel/boards/sof_sdw_max98373.c

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: GPL-2.0
+// Copyright (c) 2020 Intel Corporation
+
+/*
+ *  sof_sdw_max98373 - Helpers to handle 2x MAX98373 codec devices
+ *  from generic machine driver
+ */
+
+#include <linux/device.h>
+#include <linux/errno.h>
+#include <sound/soc.h>
+#include <sound/soc-acpi.h>
+#include "sof_sdw_common.h"
+#include "../../codecs/max98373.h"
```

Comment:
this header doesn't seem to be needed

Reply:
"../../codecs/max98373.h" can be removed, but i'm observing issue if other header files are removed, Post ryan re-spins the patch shall re-check and correct it.